popde/adaptive_kde.py

```diff
@@ -82,6 +82,33 @@ def set_per_point_bandwidth(self, pilot_values):
         self.set_bandwidth(
             self.global_bandwidth / self._local_bandwidth_factor(pilot_values)
         )
+    
+     def set_alpha(self, new_alpha):
+        """
+        Set the adaptive parameter alpha to a new value.
+
+        Parameter:
+            new_alpha (float): The new value for the adaptive parameter alpha.
+        """
+        self.alpha = new_alpha
```

Comment:
here, don't we also want to redo the calculation of the bandwidths to account for the change in alpha?  Otherwise we would have an object where the bandwidths are still the old ones

Reply:
(Specifically, this method would be used if only alpha is changed but not the global bw, so the pilot values would remain the same.)

Reply:
fixed,